src/draft/NSG/Agent/LegalEntity/NonListedCompany/BeneficialOwners.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from typing import List
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class ShareSeriesClass(str, Enum):
+    A = "A"
+    B = "B"
+    C = "C"
+    D = "D"
+    E = "E"
+
+
+class Ownerships(CamelCaseModel):
+    share_series_class: ShareSeriesClass = Field(
+        ...,
+        title="Share series class",
+        description="The class of the share series that the shareholder owns.",
+        example=ShareSeriesClass.B,
+    )
+    quantity: int = Field(
+        ...,
+        title="Quantity",
+        description="The number of shares that the shareholder owns a share series",
+        example=20,
+    )
+
+
+class ShareSeries(CamelCaseModel):
+    share_series_class: ShareSeriesClass = Field(
+        ...,
+        title="Share series class",
+        description="The type of the share series of a company",
+        example=ShareSeriesClass.A,
+    )
+    number_of_shares: int = Field(
+        ...,
+        title="Number of shares",
+        description="The total number of the shares in the share series class",
+        example=1000,
+    )
+    votes_per_share: int = Field(
+        ...,
+        title="Votes per share",
+        description="The number of votes per share in the share series",
+        example=1,
+    )
+
+
+class Shareholders(CamelCaseModel):
```

Comment:
I think this class should be called `Shareholder`, as it represents just one shareholder (we use lists of them in the response).